shared/src/test/scala/singleton/ops/ImplicitConversionSpec.scala

```diff
@@ -11,5 +11,5 @@ class ImplicitConversionSpec extends Properties("ImplicitConversion") {
   property("Wrong OpToOp") = {illTyped("""val ret : 2 + 1 = implicitly[4 - 2]"""); true}
   property("Wrong OpToSingleton") = {illTyped("""val ret : 3 = implicitly[4 - 2]"""); true}
   property("Wrong SingletonToOp") = {illTyped("""val ret : 2 + 1 = 4"""); true}
-  property("ValueOf[Op]") = wellTyped {val ret : 3 = valueOf[4 - 1]}
-}
\ No newline at end of file
+  property("ValueOf[Op]") = wellTyped {val ret : 3 = valueof.valueOf[4 - 1]}
```

Comment:
I don't want to burden you with the conflicts. I can merge #33 right now. It's ready. 
If most of the changes were done to satisfy `valueOf`, then indeed we should get rid of it before. 
Your choice.